stack-lts-16.31.yaml

```diff
@@ -0,0 +1,24 @@
+resolver: lts-16.23
+
+packages:
```

Comment:
Is there a globbing option like `- essence-of-live-coding-*`?

Reply:
I searched for it but couldn't find anything on the documentation. Trying `- essence-of-live-coding-*` does not work...

Reply:
Ok, that's still fine :)